src/features/SignInUpPages/SignInPage/SignInPage.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+import CustomInput from '../../../components/CustomInput/CustomInput';
+import CustomButton from '../../../components/CustomButton/CustomButton';
+import Spinner from '../../../components/Spinner/Spinner';
+import { userDataSelector } from '../../../redux/userAccounts/userAccount.selectors';
+import { logIn, loginChanged, passwordChanged } from '../../../redux/userAccounts/userAccount.actions';
+import { localStorageObjects, propTypesShapes } from '../../../constants';
+import Api from '../../../helpers/Api';
+import styles from '../SignInUpPage.module.scss';
+
+const SignInPage = ({ history, loginChanged, passwordChanged, logIn, userData, isUsersLoading }) => {
+  const [errors, updateErrors] = useState('');
+
+  const submitHandler = (event) => {
+    event.preventDefault();
+
+    updateErrors(() => '');
+
+    const { login, password } = userData;
+
+    if (!login.trim() || !password.trim()) {
+      updateErrors(() => 'Enter login and password.');
+    } else {
+      Api.logIn(userData)
+        .then((response) => {
+          if (!response.data.length) {
+            updateErrors(() => 'Invalid login or password');
+          } else {
+            localStorage.setItem(localStorageObjects.CURRENT_USER, JSON.stringify(userData));
+
+            logIn();
+
+            history.push('/home');
+          }
+        })
+        .catch(() => updateErrors(() => 'Server error occurred. Try again later.'))
```

Comment:
why you use .then, .catch instead async/await? 
you already changed some code to async/await instead then/catch